scripts/monitor-bb-sd.js

```diff
@@ -0,0 +1,61 @@
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
+
```

Comment:
I think this should be moved out of `scripts` to its own dir, to be separated from user scripts. An alternative could be a different subdir, like `scripts/images-updater` or something like that.

Reply:
Yes, I agree. I've been playing with the same idea myself.

Reply:
Moved to `/scripts/linkbot`